.github/workflows/tests.yml

```diff
@@ -16,10 +16,10 @@ jobs:
       TERM: dumb
     steps:
     - uses: actions/checkout@v2
-    - uses: mstksg/get-package@v1
+    - uses: delgurth/get-package@v3
```

Comment:
Any specific reason the replace the GitHub Action here?

Reply:
Yes, there was a specific reason, v1 installed packages twice on newer Ubuntu images, once via brew and once via the normal apt-get. https://github.com/mstksg/get-package/issues/5